internal/graph/check.go

```diff
@@ -123,7 +122,6 @@ type checkOutcome struct {
 }
 
 type LocalChecker struct {
-	ds                 storage.RelationshipTupleReader
```

Comment:
Not needed as we contextuall provide the datastore per ResolveCheck. Each time ResolveCheck is invoked we may be dealing with a) different contextual tuples and thus a CombinedTupleReader with a different set of tuples from the parent Check or b) a new incoming request which implies that we need to had a BoundedConcurrencyTupleReader that resets at the per-request Check concurrency limit.